### ISSUE TITLE
when no CUDA, it takes ages to run twenty newsgroup example

### ISSUE THREAD

**Issue by elisehuard** [cemoody/lda2vec#47]
Without CUDA, it's absolutely single-threaded, and that means that even on my machine which is reasonably recent with Intel Core i7 processor, it takes about 5 days to run the 200 epochs.

**Comment by radekrepo** [comment]
That's correct. Same here. Everything hangs on CUDA to pass on the computation-intensive tasks to GPU for faster computing. I had found it impossible to enable multithreaded + GPU computation with lda2vec on my Windows computer. Others say that in a Linux-based system and/or in a distributed computing cloud you can correctly install all the dependencies.

**Comment by AdrianTudC** [comment]
What is the issue here? That it is not optimized? As far as i know it's normal for a non-cuda training to take several days on a CPU.

**Comment by un-lock-me** [comment]
how you guys get the result,
I am confusing how can I run the code to see the output. except in examples/twenty_newsgroup/lda2vec/lda2vec_run.py which I got the error for cuda environment setup, most files run successfully. I want to apply on another dataset and see the result, would you please tell which files should I run in order?

Thanks:)

**Comment by acdf-pue** [comment]
Hi you all!
I'm also having problems setting up the CUDA environment. Can someone help us with that issue?

that's what a get by after installing: pip instal chainer==1.5.1

Thank you in advance..It's really important to me make lda2vec work in my system...

  File "lda2vec_run.py", line 23, in <module>
    cuda.get_device(gpu_id).use()
  File "/home/xxxx/.conda/envs/entorno/lib/python2.7/site-packages/chainer/cuda.py", line 159, in get_device
    check_cuda_available()
  File "/home/xxxx/.conda/envs/entorno/lib/python2.7/site-packages/chainer/cuda.py", line 82, in check_cuda_available
    raise RuntimeError(msg)
RuntimeError: CUDA environment is not correctly set up
(see https://github.com/pfnet/chainer#installation).cannot import name core

**Comment by AdrianTudC** [comment]
Well obviously you didn't set up your CUDA environment correctly. First set it up correctly and then lda2vec_run.py will work correctly.
